Assets/Scripts/Dialogue/DialogueManager.cs

```diff
@@ -10,6 +10,8 @@
 public class DialogueManager : Manager<DialogueManager> {
     private const string SPEECH_BUBBLE_TAG = "Speech bubble";
 
+	public const float clickDelay = 0.3f;
```

Comment:
Let's make this private, since we shouldn't need to access it elsewhere, and match case with our other constants (i.e. CLICK_DELAY).